src/components/RewardsSection/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import useEmblaCarousel from 'embla-carousel-react'
+import styles from './RewardsSection.module.scss'
+import RewardsCard, { RewardsCardProps } from '@/components/RewardsCard'
+import ArrowCircularButton from '../ArrowCircularButton'
+
+export interface RewardsSectionProps {
+  rewards: RewardsCardProps[]
+  Link: React.ComponentType<{ children: React.ReactNode; href: string }>
```

Comment:
It would be better to just type this as `any` if this is forcing you to create an entirely new component wrapping `Link` on the store

Reply:
This goes agains what we are doing related with to testing to improve our code quality.
What happens if someone passes an object, number or anything to it?

As you can see on the section, this type is the correct one for accepting a Link from nextJS so I rather keep it otherwise TS makes no point.